vaas-app/src/vaas/settings/base.py

```diff
@@ -169,6 +169,9 @@
 CELERY_IGNORE_RESULT = env.bool('CELERY_IGNORE_RESULT', False)
 CELERY_TASK_PUBLISH_RETRY = env.bool('CELERY_TASK_PUBLISH_RETRY', True)
 
+# 5min we will wait for kill task
+CELERY_TASK_SOFT_TIME_LIMIT = 300
```

Comment:
I would suggest you, to rename this variable to `CELERY_TASK_SOFT_TIME_LIMIT_SECONDS`

Reply:
👍